INFO     lingkuan.conftest:conftest.py:62 [2025-09-01 15:25:49] 用例 test_vps/test_lianxi.py::TestVPSOrderType::TestMT4ExternalOrderClose::test_trader_orderclose2 开始登录，环境: uat
INFO     lingkuan.conftest:conftest.py:100 UAT登录尝试 1/5，MFA验证码: 269107
INFO     lingkuan.conftest:conftest.py:124 UAT登录成功（第1次尝试）
INFO     lingkuan.commons.variable_manager:variable_manager.py:111 [2025-09-01 15:25] 运行时变量已保存到: D:\pycharm_test\lingkuan\VAR\runtime_vars_vps.json
INFO     lingkuan.conftest:conftest.py:152 切换到VPS URL: https://39.101.181.190/vps
INFO     lingkuan.commons.api_base:api_base.py:152 [2025-09-01 15:25:51] POST请求（JSON）: /subcontrol/trader/orderClose | 数据: {'flag': 0, 'intervalTime': 0, 'num': '1', 'closeType': 1, 'remark': '', 'symbol': 'XAUUSD', 'type': 0, 'traderId': 7305, 'account': '300151'}
INFO     lingkuan.commons.api_base:api_base.py:160 [2025-09-01 15:25:51] 请求后等待 1 秒
ERROR    lingkuan.commons.api_base:api_base.py:423 [2025-09-01 15:25:52] JSON断言失败: Failed: 响应msg字段应为success（JSON路径值不匹配） | 路径: $.msg | 响应: {"code":500,"msg":"300151暂无可平仓订单","data":null}
INFO     lingkuan.conftest:conftest.py:157 [2025-09-01 15:25:52] 用例 test_vps/test_lianxi.py::TestVPSOrderType::TestMT4ExternalOrderClose::test_trader_orderclose2 执行完毕
INFO     lingkuan.commons.variable_manager:variable_manager.py:111 [2025-09-01 15:25] 运行时变量已保存到: D:\pycharm_test\lingkuan\VAR\runtime_vars_vps.json